api/tests/util/test_detailed_formatter.py

```diff
@@ -0,0 +1,66 @@
+import json
+import logging
+from dataclasses import dataclass
+from typing import List, Dict
+
+import pytest
+
+from src.util.detailed_formatter import DetailedFormatter
+from tests.helpers import assert_all_match
+
+
+class RecordingLogHandler(logging.Handler):
+    """Log handler that records """
+
```

Comment:
The extra line between the annotation and the constructor is inconsistent with our other annotations. Less importantly, I don't feel like the annotation adds anything - it's a restatement of the class name.

Reply:
Yup, this should be removed